src/proto/type.proto

```diff
@@ -51,3 +51,7 @@ enum TTLType {
     kAbsOrLat = 5;
 }
 
+enum CompressType {
+    kNoCompress = 0;
+    kSnappy = 1;
```

Comment:
Can SQL Engine work with `CompressType`?

Reply:
can not set CompressType in sql create statement currently